nixos/modules/services/web-apps/nextcloud.nix

```
@@ -482,7 +482,7 @@ in {
           before = [ "phpfpm-nextcloud.service" ];
           path = [ occ ];
           script = ''
-            chmod og+x ${cfg.home}
+            install -m750 --owner nextcloud --group nextloud -d ${cfg.home}
```

```suggestion
            install -m750 --owner nextcloud --group nextcloud -d ${cfg.home}
```
(typo "nextloud")
Also, this doesn't actually make the home world-readable, but group-readable? I think it makes sense to do so, but the commit message is misleading.